dev-lang/xa65/xa65-2.3.8.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="An open-source 6502 cross assembler"
+DESCRIPTION="xa is a high-speed, two-pass portable cross-assembler. It \
+understands mnemonics and generates code for NMOS 6502s (such as 6502A, \
+6504, 6507, 6510, 7501, 8500, 8501, 8502 ...), CMOS 6502s (65C02 and \
+Rockwell R65C02) and the 65816."
+HOMEPAGE="https://www.floodgap.com/retrotech/xa/"
+COPYRIGHT="1989-2017 Andre Fachat
+	Jolse Maginnis
+	David Weinehall
+	Cameron Kaiser"
+LICENSE="GNU LGPL v2"
```

Comment:
It's [the GPL, not LGPL][0]

```diff
-LICENSE="GNU LGPL v2"
+LICENSE="GNU GPL v2"
```

[0]: https://www.floodgap.com/retrotech/xa/COPYING.txt

Reply:
Will fix :)